### DISCUSSION TITLE
Request for Immediate Refund and Disabling of Pro Versions

### DISCUSSION THREAD

**Discussion by harshalbhangale** [vercel/community#7010] [Help]
### Summary

Dear Vercel Support Team,
My name is Harshal Bhangale, and I am writing to request an immediate refund for two charges of $20 each that were applied to my credit card without prior notice.

While deploying an application from a GitHub organization for the first time, I created two teams and attempted to deploy two GitHub repositories. I added my credit card for what I believed to be a $0 transaction. However, I received messages indicating autopay transactions of 2000 USDC each. I immediately canceled these transactions through my credit card. Despite this, I was charged $20 twice.

Here are the links associated with the charges:

https://vercel.com/propmastr
https://vercel.com/propmastr-frontend
I kindly request that you:

Refund the $20 x 2 charges to my credit card immediately.
Disable the pro versions associated with the above links.
Attached are the receipts I received after being charged.

I appreciate your prompt attention to this matter. I was not informed about these charges beforehand, and I hope to resolve this issue quickly and amicably.

Please help me , I am new to the platform and doesn't really wanted to this transaction. Just added my card to access the platform as we do on AWS for free access .. But I was charged without being informed.Please Help

### Example

Please help me ... I don't know how to contact and request refund

### Steps to Reproduce

Please help
[Receipt-2199-1030.pdf](https://github.com/vercel/community/files/15469208/Receipt-2199-1030.pdf)
[Receipt-2038-9318.pdf](https://github.com/vercel/community/files/15469209/Receipt-2038-9318.pdf)

**Comment by amyegan** [comment]
Thanks for reaching out, @HarshalBhangale. I'm not able investigate billing issues from this public forum, but one of our billing experts will be able to look into this situation if you [submit a case](https://vercel.com/help). 

To open a new case go to [vercel.com/help](https://vercel.com/help), scroll down to the _Get Help_ form, and choose `Invoice Enquiry` as the _Problem Area_ when filling in the form.
